src/components/common/MuiMarkdown.tsx

```diff
@@ -2,8 +2,17 @@ import React from "react";
 import { Link, Typography } from "@material-ui/core";
 import Markdown from "markdown-to-jsx";
 
-class MuiMarkdown extends React.PureComponent {
+interface IMuiMarkdownProps {
+  markdown: string;
+}
+
+class MuiMarkdown extends React.PureComponent<IMuiMarkdownProps, {}> {
   render = () => {
+    const frontMatterRegExp = /(\-\-\-(.|\n)*\-\-\-)/;
```

Comment:
Shouldn't we strip this out during build time? :)

Reply:
I agree.. let me dive into WebPack once more..

Reply:
one done in the latest commit 👍